tools/scripts/features/frcnn/modeling_frcnn.py

```diff
@@ -0,0 +1,2072 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
For this task this looks good to add it here. 

We need to think how to add this functionality as an encoder (similar to encoders here : [link](https://github.com/facebookresearch/mmf/blob/master/mmf/modules/encoders.py#L194) ), so that this frcnn encoder can take an input image and return the feature directly to a multimodal model. That will enable us to run those models using this frcnn encoder end to end, which will be part of the next milestone 2 task. Something to keep in mind.